x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -50,6 +50,7 @@ function formatString(value?: string | null) {
 }
 
 const ToolTipWrapper = euiStyled.span`
+  line-height: 0px;
```

Comment:
Hmm.. This seems a bit fishy. Wouldn't be surprised if this is handled differently across browsers.